01-17-crafting-system/crafting/CraftingStation.gd

```diff
@@ -0,0 +1,57 @@
+extends Node
+class_name CraftingStation
+"""
+Responsible to craft items based on recipes
+"""
+
+
+export var craftable_items = []
+
+
+func can_craft_item(itens : Array, recipe : Array) -> int:
+	"""
+	Returns the amount of a certain item that can be crafted given a recipe and an array of itens
+	"""
+	var amount : = 999999
```

Comment:
Set the amount to `0` and remove `can_craft`. If the amount is 0, you can't craft anything. It's weird to have this function return a negative amount.